packages/backend/src/utils/npm.ts

```diff
@@ -120,6 +109,50 @@ class Command {
     }
   }
 
+  private async shouldBeUpdated(packageJson: any): Promise<boolean> {
+    const queryUrl = this.getSingleGenQueryURL(packageJson.name);
+    const npmJsModules = await npmFetch.json(queryUrl);
+    const npmJsModule: any = _.get(npmJsModules, "objects.[0]");
+    return npmJsModule ? npmJsModule.package.version !== packageJson.version : false;
```

Comment:
```suggestion
    return npmJsModule ? npmJsModule.package?.version !== packageJson.version : false;
```

Reply:
it is always exists here

Reply:
but it is not covered by type definition. seems like can get exception in runtime